### PR TITLE
Added 'Linux CLI Quickstart Guide' section to README.md

### PR DESCRIPTION
**Pull request by tznind**
This PR is to update our documentation on installing/testing RDMP especially in Linux.

I've added a 'Quick start instructions' for testing RDMP using a docker image for SqlServer but not RDMP itself.  This should be pretty easy for people to modify if they already have an sql server they want to work with (e.g. a production/testing server).

I know @rkm created a docker image for the whole RDMP CLI but it looks a bit out of date (it seems to talk about `netcore 2.2 ` for one).  Any ideas @jas88 on how we could update this docker compose and keep it current (e.g. make it part of CI)?

```
thomas@thomas-CF-31WBLAHLM:~/RDMP$ git grep docker
Dockerfile:COPY ./scripts/install-rdmp-docker.sh .
Dockerfile:RUN ./install-rdmp-docker.sh
Dockerfile:RUN rm ./install-rdmp-docker.sh
docker-compose.yml:# docker run -p 1433:1433 rdmp
```

At the very least if we have a docker image we should describe in readme how to use it (in the same way I have done in [68ccb08](https://github.com/HicServices/RDMP/pull/1263/commits/68ccb0844431eca191b409f0f6f5fd49c2782598))

### CODE REVIEW

**Comment by jas88** [comment]
@tznind WIP now - added Github action to build a Docker image with MSSQL+RDMP-CLI in, just need to fix up the timing so MSSQL is up and running when RDMP tries installing.

**Comment by manics** [comment]
Thanks for adding the Linux instructions! `libgdiplus` seems to be a runtime dependency on Ubuntu 20.04, without it I wasn't able to use https://github.com/HicServices/RDMP/releases/download/v7.0.14/rdmp-cli-linux-x64.zip

I also had to use `./rdmp gui --usc`, to get the text GUI to work properly.

FYI my test image https://gist.github.com/manics/2ad0c0e3c3c916907e32e9c0bc1ed396

**Comment by jas88** [comment]
@manics Yes - the libgdiplus dependency is a nasty legacy of accidentally relying on Windows GDI32 HBITMAP objects for icon handling (which in turn needs the Windows GDI compatibility hack for Linux, and means it can't run at all on MacOS), and accessing the icons even in text mode :( I've got a Docker build on this PR which includes the dependency and pre-configures the database too - I just need to sort out a Github permissions issue with the container repo (GHCR) and it'll be public for easier use.

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+name: Build RDMP Docker Image
+on:
+  push:
+
+jobs:
+  build:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+
```
Comment (manics) [inline .github/workflows/docker.yml:9]:
```suggestion
    permissions:
      contents: read
      packages: write

```
Dunno if this helps..... is the org set to default to read-only on the CI token?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        uses: docker/metadata-action@v4
+        with:
+          images: rdmpcli
+      - name: Push mini-container to GitHub Container Repository
+        id: push-mini-to-ghcr
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: rdmpcli
+          tags: rdmpcli:latest
+          registry: ghcr.io/hicservices
```
Comment (manics) [inline .github/workflows/docker.yml:68]:
```suggestion
          image: ghcr.io/${{ github.repository }}:latest
```

**Review by manics — COMMENTED** [review]
Not sure if any of these suggestions will help.... but they're based on one of my repos where I'm building with Docker and pushing to GHCR:
https://github.com/manics/binderlite-builder/blob/fafa60c6882f7af22da83b4526ca8f58a44c5038/.github/workflows/build.yml#L29-L50

**Comment by jas88** [comment]
@manics Got it fixed thanks - turns out the packaged Github action was trying to talk to Docker Hub for something and failing (?!) - switching to plain "podman push ghcr.io/hicservices/rdmpcli:latest" fixed it. Will be adding instructions and merging soon now!

**Comment by jas88** [comment]
@tznind Almost set to go - I'll clean up those files then merge if that's OK?

**Comment by tznind** [comment]
Yup, fire away!

**Pull request merged**